### PR TITLE
removing content

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**What**
Removing existing content from the templates

@mathana96

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
Hi @mathana96. 
Steps to review:
Check that there are no errors in the code changes.
Ensure that there are no spots where I missed any content. :)

File: mentors/index.md [hunk 1]
```diff
@@ -5,18 +5,11 @@ title: Mentors
...
-![Picture](https://avatars3.githubusercontent.com/u/981838?v=3&s=160)
 
-[twitter/pantroken](https://twitter.com/pantroken)
+## Niall Kane
 
-## Luigi Zuccarelli
-![Picture](https://avatars0.githubusercontent.com/u/12412882?v=3&s=160)
 
-[github/luigizuccarelli](https://github.com/luigizuccarelli)
+## Matt Sreedaran
```
Comment (mathana96) [inline mentors/index.md]: Mathana Sreedaran (Matt)
^ A rare occasion where my actual first name is used :D

File: index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: default
 title: Home
 ---
 
-Welcome to CoderDojo Ardkeen Waterford
+Welcome to Waterford Coderdojo
```
Comment (mathana96) [inline index.md]: Coderdojo Waterford maybe?

File: index.md [hunk 3]
```diff
@@ -38,4 +38,4 @@ See [projects](/project) page for more details.
 <hr>
 
 <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">
-<img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a> Coder Dojo Ardkeen website is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons License</a>.
\ No newline at end of file
+<img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a> Waterford Coderdojo website is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons License</a>.
```
Comment (mathana96) [inline index.md]: Coderdojo Waterford #2

**Review by mathana96 — CHANGES_REQUESTED** [review]
My first review :o

**Comment by laurafitzgerald** [comment]
@mathana96 changes made as requested.

**Review by mathana96 — APPROVED** [review]
Thank you!

**Comment by laurafitzgerald** [comment]
@mathana96 merging, thanks.

**Pull request merged**